bin/release.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+error_missing_field () {
+  echo "Missing required env var: $1"
+  exit 1
+}
+
+# Ensure all required variables are set before doing any work
+if [[ -z ${GITHUB_USER:-} ]]; then error_missing_field 'GITHUB_USER'; fi
+if [[ -z ${GITHUB_ACCESS_TOKEN:-} ]]; then error_missing_field "GITHUB_ACCESS_TOKEN"; fi
+if [[ -z ${RELEASE_BRANCH:-} ]]; then error_missing_field "RELEASE_BRANCH"; fi
+if [[ -z ${VERSION:-} ]]; then error_missing_field "VERSION"; fi
+if [[ -z ${AWS_ACCESS_KEY_ID:-} ]]; then error_missing_field "AWS_ACCESS_KEY_ID"; fi
+if [[ -z ${AWS_SECRET_ACCESS_KEY:-} ]]; then error_missing_field "AWS_SECRET_ACCESS_KEY"; fi
+if [[ -z ${AWS_SESSION_TOKEN:-} ]]; then error_missing_field "AWS_SESSION_TOKEN"; fi
+
+# Set defaults for CDN upload if not set
+: "${BUCKET_NAME:=bugsnagcdn}"
```

Comment:
Wasn't sure if this is how we wanted to handle these variables for the CDN upload or make them mandatory like the ones above

Reply:
🤔 good question. I'm up for keeping _anything_ related to our AWS instances out of code